automation/src/test/scala/org/broadinstitute/dsde/workbench/GeneratedLeonardoClient.scala

```diff
@@ -53,6 +53,12 @@ object GeneratedLeonardoClient {
       api <- IO(new DisksApi(apiClient))
     } yield api
 
+  def generateAppsApi()(implicit accessToken: IO[AuthToken]): IO[AppsApi] =
```

Comment:
Added this function in order to avoid using the [legacy wrapper](https://github.com/DataBiosphere/leonardo/blob/develop/automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoApiClient.scala).